qutip/bloch.py

```diff
@@ -697,19 +714,21 @@ def plot_vectors(self):
             zs3d = self.vectors[k][2] * array([0, 1])
 
             color = self.vector_color[mod(k, len(self.vector_color))]
+            alpha = self.vector_alpha[mod(k, len(self.vector_alpha))]
```

Comment:
We shouldn't apply a `mod(k, ...)` here. It is the correct thing to do for the color because those are expected to cycle through a list of colors, but for the alpha values we have an explicit alpha value for every vector, so there should be no need to wrap.